### DISCUSSION TITLE
[IntelliSense] Feature request: `tailwindCSS.experimental.configFile` to extend the default

### DISCUSSION THREAD

**Discussion by domdomegg** [tailwindlabs/tailwindcss#13590] [Ideas]
In most cases, the intellisense plugin finds my tailwind config file just fine!

However, there's a couple exceptions where I'd like to override it with `tailwindCSS.experimental.configFile`. However if I do this I then need to specify all the config files manually.

I've got a monorepo setup with many projects using tailwind, and having to add each one manually is quite tedious (especially as it's easy to forget when creating a new project and get confused).

It'd would be great if either:
- If no config file is found in `tailwindCSS.experimental.configFile`, it uses the default logic lookup
- A new option like `tailwindCSS.experimental.configFile.extend` is added that has this behaviour

**Comment by EKliptus** [comment]
Exact same need,
I have a monorepo with :
- base folder where I share components
- sites folder with many astro sites project

By default tailwind detect all my astro project config files but I have no intellisense in my base/src/**
So I must use `tailwindCSS.experimental.configFile` to tell my base folder exist. 
```
  "tailwindCSS.experimental.configFile": {
    "base/src/styles/tw-import.css": "base/src/**",
  },
  ```
  This precedent config crushes the automatic detection behavior and my astro project isn't founded !
  
  a `tailwindCSS.experimental.configFile.extend` would be a great evolution !
  
  or with wilcards like this :
  ```
    "tailwindCSS.experimental.configFile": {
    "base/src/styles/tw-import.css": "base/src/**",
    "sites/**/src/styles/global.css": "sites/**/src/**"
    // or :
    "sites/**/astro.config.mjs": "sites/**/src/**"
  },
  ```

Thanks
